### DISCUSSION TITLE
Is it possible to create a project with only serverless/edge functions?

### DISCUSSION THREAD

**Discussion by isaacharrisholt** [vercel/community#2474] [Help] [answered]
### Summary

Hi all,

My team and I currently maintain 3 separate but interconnected apps on Vercel, and there are some API routes that we want to be able to share between the apps.

I was wondering if it's possible to have a project that only hosts serverless/edge functions for use as an API, without needing a frontend of any sort. Extra points if we're able to mix and match languages, as we currently have a few AWS lambda functions in Go and Python that we'd like to migrate to Vercel to reduce deployment headaches.

Thanks in advance!

**Comment by amyegan** [comment, marked as answer]
Good question, @isaacharrisholt! Yes, you can have project that only hosts serverless and edge functions without requiring a frontend. You can mix the available serverless runtimes in the `/api` directory. I know there are some people who use both JS and Python functions, for example. More info is available here:
- [Serverless Function Runtimes](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes#)
  - [Go](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/go)
  - [Python](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/python)

**Reply by isaacharrisholt** [reply]
Hi @amyegan! Thanks for replying so quickly. How do you configure Vercel to do this? When I tried, it seemed to need the build command to succeed. In my case though, there would be no build command. Can I just set that as an empty field?
